tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/FundingSourceControllerTests.cs

```diff
@@ -18,17 +16,86 @@
 namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Controllers
 {
     [TestFixture]
-    public sealed class FundingSourceControllerTests
+    [Parallelizable(ParallelScope.All)]
+    internal sealed class FundingSourceControllerTests
     {
         [Test]
-        public void Constructor_NullOrderRepository_ThrowsException()
+        public void Constructor_NullOrderRepository_ThrowsArgumentNullException()
         {
             var builder = FundingSourceControllerBuilder.Create()
                 .WithOrderRepository(null);
 
             Assert.Throws<ArgumentNullException>(() => builder.Build());
         }
 
+        [Test]
+        public async Task GetAsync_OrderDoesNotExist_ReturnsNotFound()
+        {
+            const string orderId = "DOESNOTEXIST";
+            var context = FundingSourceControllerTestContext.Setup();
+            context.Order = null;
+
+            using var controller = context.Controller;
+
+            var result = await controller.GetAsync(orderId);
+
+            result.Result.Should().BeOfType<NotFoundResult>();
+        }
+
+        [Test]
+        public async Task GetAsync_DifferentOrganisationId_ForbiddenReturned()
+        {
+            const string orderId = "C0000014-01";
+            var context = FundingSourceControllerTestContext.Setup();
+            context.PrimaryOrganisationId = Guid.NewGuid();
+            context.Order = OrderBuilder
+                .Create()
+                .WithOrganisationId(Guid.NewGuid())
+                .Build();
+
+            using var controller = context.Controller;
+            var actual = await controller.GetAsync(orderId);
+
+            actual.Result.Should().BeOfType<ForbidResult>();
+        }
+
+        [Test]
+        public async Task GetAsync_OrderExists_FundingSourceDetailsReturned()
+        {
+            const string orderId = "C0000014-01";
+
```

Comment:
Can you take away this new line, so it matches the pattern of the other tests you have done